### PR TITLE
Add basic map of Nigeria to personal page

### PR DESCRIPTION
**Pull request by oolusina**
I got an alert about publishing a file on github that has an API key in it. Is this wrong to do?

### CODE REVIEW

**Comment by abjindal-sps** [comment]
> I got an alert about publishing a file on github that has an API key in it. Is this wrong to do?

Yes, it is not a general practice to publish an API key on github as it can be reused/misused by reader of the API key, you can redact it before sending in a code review/pushing to git.  usually you can have a shell variable on your system for your api keys etc. and have that variable be passed as a parameter into your application.

**Comment by oolusina** [comment]
Could not figure out how to pass in the API key from the environment variable i created, but i reset it and redacted it from this commit.

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -12,6 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+function createMap() {
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 9.082, lng: 8.675}, zoom: 5});
```
Comment (abjindal-sps) [inline portfolio/src/main/webapp/script.js:18]: nit: Ideally you should define these as constant variables in some central place.

**Review by abjindal-sps — APPROVED** [review]

**Review by abjindal-sps — APPROVED** [review]

**Comment by abjindal-sps** [comment]
That's ok for now :)

**Pull request merged**